serialize.js

```diff
@@ -33,6 +33,8 @@ function serializeElement(elem) {
             strings.push.apply(strings, elem.childNodes.map(serializeNode))
         } else if (elem.textContent || elem.innerText) {
             strings.push(escapeText(elem.textContent || elem.innerText))
+        } else if (elem.innerHTML) {
+            strings.push(elem.innerHTML)
```

Comment:
For html injection, as is mentioned in https://github.com/Matt-Esch/virtual-dom/blob/master/docs/vtext.md#html-injection